ex/high_throughput_inference/mock_app.py

```diff
@@ -41,16 +41,23 @@
 import os
 import time
 import torch
-import numbers
 
-from collections import OrderedDict
+from mpi4py import MPI
 from smartsim._core.mli.message_handler import MessageHandler
 from smartsim.log import get_logger
+from smartsim._core.utils.timings import PerfTimer
+
+torch.set_num_interop_threads(16)
+torch.set_num_threads(1)
 
 logger = get_logger("App")
 
+CHECK_RESULTS_AND_MAKE_ALL_SLOWER = False
+
 class ProtoClient:
     def __init__(self, timing_on: bool):
+        comm = MPI.COMM_WORLD
+        rank = comm.Get_rank()
         connect_to_infrastructure()
         ddict_str = os.environ["SS_DRG_DDICT"]
```

Comment:
per my PR review, we're planning on renaming `SS_DRG_DDICT` to `SS_INFRA_BACKBONE`

Reply:
Addressing this in the merger